packages/app/src/lib/tooltip.tsx

```diff
@@ -1,23 +1,44 @@
 import Tippy, { TippyProps } from '@tippyjs/react';
 import { Instance } from 'tippy.js';
 import 'tippy.js/dist/tippy.css';
-import 'tippy.js/themes/light.css';
+import styled from 'styled-components';
+import css from '@styled-system/css';
+import { useBreakpoints } from '~/utils/use-breakpoints';
+
 import { isDefined } from 'ts-is-present';
 
 let handleMount: undefined | ((tippyInstance: Instance) => void);
 
+const StyledTippy = styled(Tippy)(
+  css({
+    backgroundColor: 'white',
+    color: 'black',
+    boxShadow: 'tile',
+    '.tippy-arrow': {
+      color: 'white',
+    },
+  })
+);
+
 /**
  * Usage:
  *     <WithTooltip content={<p>message</p>}><button>foo</button></WithTooltip>
  */
 
-export function WithTooltip(props: TippyProps) {
+export function WithTooltip(props: Omit<TippyProps, 'theme'>) {
+  const breakpoints = useBreakpoints(true);
+
   if (!isDefined(props.content)) {
     return <>{props.children}</>;
   }
 
   return (
-    <Tippy theme="light" appendTo={getBody} onMount={handleMount} {...props} />
+    <StyledTippy
+      appendTo={getBody}
+      onMount={handleMount}
```

Comment:
while you're at it, could you fix another issue with the tooltip which is probably fixed with this change:
```suggestion
      {...handleMount ? { onMount: handleMount } : {}}
```

I've assigned you on trello to the related ticket ("Tippy cannot read property apply of undefined")

Reply:
Hmm I'm a bit confused by this. What is this solution doing? I can't apply the spread operator on a function and I'm getting a typescript error that makes it seem like the original solution is correct 
![image](https://user-images.githubusercontent.com/75375917/126633606-588f06a5-5626-41e2-bb55-2fdf99b73616.png)

Reply:
Ah yes, a bit of explanation would have been useful.
Apparently the tippy library cannot handle an `onMount` prop with an `undefined` value. It's probably checking on existence of the `onMount` key in the props and it will use that value to overwrite some default value.. 

The `handleMount` will be `undefined` in production builds so those builds would throw an error when interacting with a tooltip. 

The proposed solution is to only apply the `onMount` prop when the `handleMount` handler is defined.

I believe the TS error is caused by the wrapping Tippy with styled-components. A fix could be to cast StyledTippy to the type of Tippy:
```tsx
const StyledTippy = styled(Tippy)`` as typeof Tippy;
```